lib/src/navbar/navbar.component.scss

```diff
@@ -63,6 +63,8 @@ $navbar-fixed-shadow: 0px 2px 6px $shadow;
         display: flex;
         justify-content: center;
         align-items: center;
+
+        &.logo-condense { padding-right: 25px !important; }
```

Comment:
Can this be removed? I didn't see any difference when removing the `!important`. I try to avoid it when possible